README.md

```diff
@@ -1,7 +1,21 @@
-# gptravel
+# GPTravel ✈️
 ![Tests](https://github.com/RobertoCorti/gptravel/actions/workflows/python-tests.yml/badge.svg)
+![GitHub Tags](https://img.shields.io/github/tag/RobertoCorti/gptravel.svg)
 
```

Comment:
@RobertoCorti  I would suggest to add also the License badge if possible. You can check this page: https://gist.github.com/lukas-h/2a5d00690736b4c3a7ba

Reply:
Ok, I'll add